imagetagger/imagetagger/annotations/static/annotations/js/annotations.js

```diff
@@ -932,6 +939,20 @@ function calculateImageScale() {
     } else {
       handleNewAnnotations();
     }
+
+  }
+
+  function loadAnnotated() {
+    let allKeys = Object.keys(gAnnotationCache);
+    for(let i = 0; i < allKeys.length; i++) {
+      console.log(gAnnotationCache[allKeys[i]]);
+      console.log(gAnnotationCache);
```

Comment:
Can we remove the logs?

Reply:
definitely! I was using log statements to test my code, forgot to get rid of them.